sycl/test/removed_api/sycl_2020_deprecated.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clangxx %fsycl-host-only -fsyntax-only -sycl-std=2020 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
```

Comment:
please rename the file to something like expected_error.cpp

Reply:
Renamed